### PR TITLE
dashboard: link docs to public site

### PR DESCRIPTION
**Pull request by dominic**

### CODE REVIEW

**Comment by jeffomatic** [comment]
I think we should use specific version links, so that your version of dashboard is synched with the version of the docs that the links lead to.

**Comment by dominic** [comment]
Right - it won't always be latest.

**Comment by dominic** [comment]
@jeffomatic PTAL again

File: dashboard/src/features/app/components/Navigation/Navigation.jsx [hunk 1]
```diff
@@ -100,11 +100,21 @@ class Navigation extends React.Component {
...
 export default connect(
-  state => ({
-    routing: state.routing, // required for <Link>s to update active state on navigation
-    showSync: state.core.configured && !state.core.generator,
-    production: state.core.production,
-  }),
+  state => {
+    let docVersion = ''
+
+    const versionComponents = state.core.version.match('^([0-9]\\.[0-9])\\.(.*)')
```
Comment (jeffomatic) [inline dashboard/src/features/app/components/Navigation/Navigation.jsx]: I suspect you don't need the second set of parentheses in the regex here.

Comment (dominic) [inline dashboard/src/features/app/components/Navigation/Navigation.jsx]: :+1:

**Pull request merged**